Makefile

```diff
@@ -1,14 +1,37 @@
-.PHONY: build run defult
+.PHONY: build run debug test doc
 
 IMAGE_NAME = modularitycontainers/memcached
+IMAGE_OPTIONS = \
+    -p 127.0.0.1:11211:11211
 
-default: run
+VERSION = 1.14
```

Comment:
since this does not build from a module, you can't be sure that you'll be getting 1.14, is that true?

Reply:
@bkabrda I wrote the version against memcached -v value. Is it correct? Like in bind case.

Reply:
I would personally prefer [the way I did it](https://github.com/container-images/tools/pull/4/files#diff-37c04ce6822c2b1d4aa6db755e21f3d2R9) -- don't mention versioning but instead use whatever the platform offers, so:

```
variants:
  upstream:
  fedora:
  centos:
  ...
```

But I would prefer discussing this.

Reply:
I guess this comes down to what the author wants/needs. In this case, I don't see `{{ spec.version }}` used anywhere, so I'm wondering why this (and the `version` section in multispec) is even necessary.

Reply:
@bkabrda @TomasTomecek New distgen version fix the problem with `--multispec-selector`.